pkg/collectors/policyreport.go

```diff
@@ -119,12 +121,15 @@ func getReports(clusterID string, pr *v1alpha2.PolicyReport) [][]string {
 		default:
 			severity = "unknown"
 		}
-		if reportResult.Policy != "" && category != "" && result != "" {
-			policy = reportResult.Policy
-			metric = append(metric, clusterID, category, policy, result, severity)
+		if reportResult.Policy != "" {
+			metric = append(metric, clusterID,
+				reportResult.Category,
+				reportResult.Policy,
+				result, severity)
```

Comment:
I'm not sure why we declare `metric` and then append to it. Might as well initialize it in one line here?
```suggestion
			metric := []string{clusterID,
				reportResult.Category,
				reportResult.Policy,
				result, severity}
```